docker/docker-bigtable/docker-compose.yml

```diff
@@ -0,0 +1,65 @@
+version: '2'
```

Comment:
by convention, enviroments that load the locally built binary should have `dev` in the name, so `docker-dev-bigtable`
it should also be mentioned in docs/docker.md